### ISSUE TITLE
Not working when changing number of line from 0 to 3

### ISSUE THREAD

**Issue by PK18AK** [null09264/FRHyperLabel#26]
Hello,

I just run the same code downloaded from your repo. its working perfectly but when I changed number of line in code than button click stop working.

Below is the code which I am using.
	

```
FRHyperLabel *label = self.label;
	label.numberOfLines = 3;

```	

```
	//Step 1: Define a normal attributed string for non-link texts
	NSString *string = @"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque quis blandit eros, sit amet vehicula justo. Nam at urna neque. Maecenas ac sem eu sem porta dictum nec vel tellus.\n Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque quis blandit eros, sit amet vehicula justo. Nam at urna neque. Maecenas ac sem eu sem porta dictum nec vel tellus.";
    
	NSDictionary *attributes = @{NSForegroundColorAttributeName: [UIColor blackColor],NSFontAttributeName: [UIFont preferredFontForTextStyle:UIFontTextStyleHeadline]};
	label.attributedText = [[NSAttributedString alloc]initWithString:string attributes:attributes];


```
```
//Step 2: Define a selection handler block
	void(^handler)(FRHyperLabel *label, NSString *substring) = ^(FRHyperLabel *label, NSString *substring){
		UIAlertController *controller = [UIAlertController alertControllerWithTitle:substring message:nil preferredStyle:UIAlertControllerStyleAlert];
		[controller addAction:[UIAlertAction actionWithTitle:@"Cancel" style:UIAlertActionStyleCancel handler:nil]];
		[self presentViewController:controller animated:YES completion:nil];
	};

```

```
	//Step 3: Add link substrings
	[label setLinksForSubstrings:@[@"Lorem"] withLinkHandler:handler];


```

**Comment by fischsux** [comment]
Thanks for the feedback. The issue is fixed in 1.0.4 :)

**Comment by Immortalpk** [comment]
Still I can see the same issue, any one can help me on this. I am using swift 4.2.
